modules/sovrnAnalyticsAdapter.js

```diff
@@ -0,0 +1,217 @@
+import adapter from 'src/AnalyticsAdapter'
+import adaptermanager from 'src/adaptermanager'
+import CONSTANTS from 'src/constants.json'
+import {ajaxBuilder} from 'src/ajax'
+import * as utils from 'src/utils'
+import {config} from 'src/config'
+
+const ajax = ajaxBuilder(0)
+
+const {
+  EVENTS: {
+    AUCTION_END,
+    BID_REQUESTED,
+    BID_ADJUSTMENT,
+    BID_RESPONSE,
+    BID_WON
+  }
+} = CONSTANTS
+
+const PBA_URL = 'http://pba-collector-adapter.us-east-2.elasticbeanstalk.com/analytics'
```

Comment:
Can we do this in the open source repo? We can make it configurable in our maestro pipeline, I believe. But, I didn't think the open source repo has env configs available for the adapters. Lets discuss.

Reply:
Yeah, we can discuss this today.

Reply:
For whatever this eventually gets set to, it seems like it might not be appropriate to use a region-specific URL. It seems like the open-source should use a painfully generic name, but anything used internally seems like it should be configurable.

Reply:
Yeah, the sovrn-pba.js pipeline that was initially setup is using `pba.lijit.com` with a certificate. We will be using that (or something similar) once the pipeline is in the new AWS production. That URL is just for the pipeline I spun up because of changes to the data introduced by the adapter.

Reply:
we can default to the prod url and allow passing in an alt url in the enableAnalytics override along with the affillateId.

Reply:
Do we know the prod url yet? And cool, I'll add an the option to pass in an alt url.

Reply:
No, not yet. Hope to have the ingest pipeline in TF and running in the next day or two. Then need to work with REO on names and certificates.